### PR TITLE
Repair rendering bug in Firefox

### PR DESCRIPTION
**Pull request by lukevmorris**
Connect #2520

### CODE REVIEW

**Comment by jaredscheib** [comment]
I don't know if we should update the changelog for now on these multitenancy fixes. Thoughts, @nhaugo?

**Comment by alexpaxton** [comment]
screen-shot-2017-12-11-at-4 10 54-pm

I'm thinking we set the height media query to be a little larger so that if the user has 2 custom links (which I think is the case for quartz users) it doesn't go off the screen. This screen is 800px tall

File: ui/src/style/layout/sidebar.scss [hunk 1]
```diff
@@ -310,3 +292,52 @@ span.icon.sidebar--icon.sidebar--icon__superadmin {
...
+.sidebar-menu--section__custom-links:after {
+  display: none;
+  border-top-right-radius: $radius;
+}
+.sidebar-menu--user-nav {
+  top: initial;
+  bottom: 0;
+}
+
+@media only screen and (min-height: 750px) {
```
Comment (alexpaxton) [inline ui/src/style/layout/sidebar.scss]: As mentioned in the comment in the PR I'd up this to `800px` to allow for at most 2 custom links to not get cropped

Comment (lukevmorris) [reply ui/src/style/layout/sidebar.scss]: Fixed!

**Pull request merged**